.github/workflows/ci.yml

```diff
@@ -111,10 +111,8 @@ jobs:
         with:
           save-if: ${{ github.event_name != 'merge_group' }}
 
-      # TODO: Add --target x86_64-unknown-none to the no_std check once we solve the compilation issues with it.
-      #       https://github.com/linebender/parley/issues/86
       - name: cargo clippy (no_std)
-        run: cargo hack clippy --workspace --locked --optional-deps --each-feature --ignore-unknown-features --features libm --exclude-features ${{ env.FEATURES_DEPENDING_ON_STD }} -- -D warnings
+        run: cargo hack clippy ${{ env.RUST_MIN_VER_PKGS }} --locked --optional-deps --each-feature --ignore-unknown-features --features libm --exclude-features ${{ env.FEATURES_DEPENDING_ON_STD }} --target x86_64-unknown-none -- -D warnings
```

Comment:
Hmm. I *think* the idiomatic approach here would be to add a new environment variable? cc @xStrom

Reply:
An even better solution is a brand new `cargo-hack` feature `--must-have-and-exclude-feature std`. Unfortunately it's been stuck in review queue for a month at [cargo-hack#262](https://github.com/taiki-e/cargo-hack/pull/262).

For now I think it's fine to abuse `${{ env.RUST_MIN_VER_PKGS }}` and just get this landed. When the proper solution becomes possible we'll adjust it.